types/eslint/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/eslint",
-    "version": "9.6.9999",
+    "version": "9.9.9999",
```

Comment:
I think it's not necessary to update the version template because the `no-void` rule has been substantially unchanged since ESLint v7.0.0-alpha.0. Here is the commit that introduced the `allowAsStatement` option: https://github.com/eslint/eslint/commit/8f1020ff711b0c57d590bf666e2841f64186d083.

Reply:
Thank you for your feedback! I’ve fixed it as suggested.